### PR TITLE
Dash CSS improvements

### PR DESCRIPTION
**Pull request by Kapeli**
TL;DR:
* Hide header
* Remove floating stuff (it's annoying and makes scrolling really slow in Dash)
* Make absolutely positioned elements static
* Increase font size a bit
* Remove borders/box shadows

### CODE REVIEW

**Comment by segiddins** [comment]
Thanks @Kapeli ! I'll review when I get to the office.

File: lib/jazzy/assets/css/jazzy.css.scss [hunk 1]
```diff
@@ -873,4 +879,24 @@ html.dash {
...
+    box-shadow: none;
+  }
+  #reference .chapter {
+    border:none;
+    box-shadow: none;
+  }
+  #footer {
+    position: static;
+  }
+}
```
Comment (segiddins) [inline lib/jazzy/assets/css/jazzy.css.scss]: please keep the trailing newline

**Comment by segiddins** [comment]
@Kapeli any chance you could share the before/after screenshots?

**Comment by Kapeli** [comment]
I've added the trailing newline back.

Before:

screen shot 2015-01-13 at 20 30 16

After:

screen shot 2015-01-13 at 20 31 22

**Comment by jpsim** [comment]
Thanks @Kapeli, nice improvements!

**Comment by Kapeli** [comment]
:+1:

**Comment by segiddins** [comment]
Wow. Thanks @Kapeli, this makes jazzy look awesome in Dash.

**Pull request merged**